src/clientdlg.cpp

```diff
@@ -1293,6 +1234,8 @@ void CClientDlg::Disconnect()
 
     // clear mixer board (remove all faders)
     MainMixerBoard->HideAll();
+
+    SetMixerBoardDeco ( RS_UNDEFINED, pClient->GetGUIDesign() );
```

Comment:
Is this needed?

Reply:
When you disconnect, you're no longer being recorded.  As far as I know, the status would have been reset and the banner already removed.  But then, the client dialog should already be handling disconnection robustly.  If there some issue with it I'm not aware of, then maybe it's needed.

Reply:
Um, depends on what `MainMixerBoard->HideAll();` does, I guess.  I'd expect it not to be, as the deco is inside what `HideAll` hides.  Ideall `HideAll` would know to tidy things up, too...